File: Entitys/Transaction.cs
using Microsoft.AspNetCore.Identity;
using System.ComponentModel.DataAnnotations;
using System.Diagnostics.CodeAnalysis;

namespace Entitys.Entity
{
    public class Transaction
    {
        [Required]
        public Guid Id { get; set; }
        [Required]
        public Account Account { get; set; }
        public TranssactionType TransactionType { get; set; }
        [Required]
        public decimal Ammount { get; set; }
        [Required]
        public DateTime Date { get; set; } = DateTime.Now;
        [Required]
        public IdentityUser OtherInvolved { get; set; }
        [AllowNull]
        public string Concept { get; set; }
    }
}

File: TrasBankCli/src/app/Models/Transaction/transaction.ts
export class Transaction {
  type: string;
  ammount: number;
  date: Date;
  other: string;
  concep: string;
  constructor(type: string, ammount: number, date: Date, other: string, concep: string) {
    this.type = type;
    this.ammount = ammount;
    this.date = date;
    this.other = other;
    this.concep = concep;
  }
}
